tasks/main.yml

```diff
@@ -52,6 +52,7 @@
         name: libselinux-python
         state: present
       when: ansible_os_family == "RedHat" and ansible_distribution_major_version >= 7
+      ignore_errors: true # The when statement above is not reliable on hLinux
```

Comment:
Would be nicer if we could improve the conditional instead.
Is this because of some facts gathering issue on hLinux? - 
What is the error you were seeing?